README.md

```diff
@@ -2,14 +2,23 @@
 
 [Production Site](https://scoopful.herokuapp.com)
 
-Scoops to Stoop: bringing the bulk aisle right to your front step. 
+Scoops to Stoop: bringing the bulk aisle right to your front step. User can login to view and check out food items by the pound in a digital grocery store bulk aisle.
 
 Eliminate food waste by ordering as much or as little as you need of an ingredient. 
 
-## Possible Extensions
-Show recipies by items ordered or order all items for a reciepe. Multiple units, such as by volume or handful. Change currency of order. 
-
 #### Project Manager 
 Amy Kintner
 #### Project Members
-Mike Schutte, Kyle Heppenstall, Annie Wolff
\ No newline at end of file
+Mike Schutte, Kyle Heppenstall, Annie Wolff
+
+Tech Stack: Rails, PostgreSQL, RSpec, Sass, Haml, Heroku, Capybara, bcrypt
+
+Follow these steps to run tests or host the site locally.
+
+1. To get started make sure you have postgres, Ruby, and rails 5  (`gem install rails`) installed on your machine.
+1. Clone the repository: `git clone https://github.com/kheppenstall/idea_box.git`
+1. `cd` into the directory `scoopful` and `bundle install` in the command line.
+1. Run `rake db:create` and `rake db:migrate` in the command line to get your schema up to date.
```

Comment:
@kheppenstall : Can you one line this to `rake db:create db:migrate db:test:prepare` ?